ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-zone/provision/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-zone-provision-chain.yaml

```diff
@@ -0,0 +1,31 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-zone-provision
+  steps: 
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-credentials-mode
+    - ref: ipi-conf-gcp-xpn-network
+    - ref: ipi-conf-gcp-create-firewall-rules
+    - ref: gcp-provision-private-dns-zone-in-service-project
+    - ref: gcp-provision-xpn-no-fw-no-dns-passthrough-serviceaccount
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+    - ref: cucushift-installer-check-gcp-credentials-mode
+  env:
+  - name: BASE_DOMAIN
+    default: "qe.gcp.devcluster.openshift.com"
```

Comment:
This domain will be set in install-config.yaml, right? That remind me `cucushift-installer-rehearse-gcp-ipi-xpn-techpreview`, in that install workflow, `BASE_DOMAIN` are set, but I did not find out where to set it in install-config.yaml.

Reply:
It's used in step `gcp-provision-private-dns-zone-in-service-project`.

Reply:
Yeah, I saw it is used in `gcp-provision-private-dns-zone-in-service-project`, where to set the domain in install-config.yaml?

Reply:
Okay, I found it, in `cucushift-installer-rehearse-gcp-ipi-xpn-techpreview`, the baseDomain is set in `ipi-conf-gcp-public-dns-zone`.

From my understanding, baseDomain is gotten from `${CLUSTER_PROFILE_DIR}/public_hosted_zone`, and set in `ipi-conf-gcp` as a default behavior, unless there is some explicit custom setting, e.g:  `ipi-conf-gcp-public-dns-zone` or `ipi-conf-gcp-basedomain`. For easier understanding, I am suggesting we can explicitly set  baseDomain in this install workflow, or update `gcp-provision-private-dns-zone-in-service-project` step to get the default baseDomain from `${CLUSTER_PROFILE_DIR}/public_hosted_zone` when "BASE_DOMAIN" is empty, so that keep consistent.

Reply:
FYI I removed the env var `BASE_DOMAIN` from the gcp provision step `gcp-provision-private-dns-zone-in-service-project`, and let it get the base domain the same way as `ipi-conf-gcp`.